tutorials/get-started-canonical-kubernetes/get-started-canonical-kubernetes.md

```diff
@@ -0,0 +1,187 @@
+ 
+Deployment will take several minutes as Juju creates new instances in the cloud and sets up the Kubernetes cluster components. Pending units are outlined in *orange*. Up and running ones are outlined in *black*.
+![Status GUI](./images/status-gui.png)
+ 
+ 
+ 
+## Getting started
+Duration: 5:00
+ 
+### Install the Juju client
+ 
+**You will need to have Juju installed locally to operate your cluster. Skip this step if you already have it.**
+ 
+Juju is available as a client on many platforms and distributions. Regardless of your operating system, getting all the required tools is very easy. Visit the [install docs](https://jujucharms.com/docs/stable/reference-install#getting-the-latest-juju) to get the latest version of Juju on **macOS**, **Windows** or **CentOS**.
+ 
+If you are running **Ubuntu**, you can install Juju through the following steps:
+ 
+1. It’s helpful to Install Snappy if you don’t have it already. 
+`$ sudo apt install snapd`
+ 
+2. Install Juju to get the command line client. 
+`$ sudo snap install juju --classic`
+ 
+3. Verify you can run Juju. You will see a summary and a list of common commands.
+`$ juju`
+ 
+You’re done! To learn more on how to get started with Juju, visit the [documentation](https://jujucharms.com/docs/stable/getting-started). 
+ 
+For general help in the CLI: 
+`$ juju help`
+ 
+ 
+### Operate your cluster with kubectl 
+ 
+To deploy and manage applications on Kubernetes you need *kubectl*. Learn more by visiting the [official documentation](https://kubernetes.io/docs/user-guide/kubectl/).
+ 
+1. For **Ubuntu** users, install the [kubectl command line](https://kubernetes.io/docs/tasks/tools/install-kubectl/): 
+`$ sudo snap install kubectl --classic`
+If you are running **macOS**:
+`$ brew install kubectl`
+For **Windows** users, use [Chocolatey](https://chocolatey.org/install):
+`$ choco install kubectl`
+For **CentOS and other Linuxes**:
+`$ sudo snap install kubectl --classic` 
+ 
+2. Verify you can run kubectl. You will see a list of commands to control your cluster.
+`$ kubectl`
+ 
+3. To connect to JAAS from the command line you'll need to register with the JAAS controller. You’ll only need to do this the first time.
+`$ juju register jimm.jujucharms.com`
+ 
+4. This command will open a new window in your browser. Use Ubuntu SSO to login and authorise your account. 
+ 
+5. You will then be asked to enter a descriptive name for the JAAS controller.  We suggest using *jaas*. 
+ 
+6. JAAS users with existing models might first need to switch to the relevant model. To see all your models:
+`$ juju models`
+
+7. To select the relevant one, run:
+`$ juju switch <model-name>`
+ 
+ 
+ 
+## Operating your Kubernetes cluster 
+Duration: 5:00
+ 
+1. Fetch the credentials from the Kubernetes Master charm. 
+`$ mkdir -p ~/.kube`
+
+2. Then run:
+`$ juju scp kubernetes-master/0:config ~/.kube/config`
+This might open a separate modal window, in which you should enter the passphrase for your SSH key.
```

Comment:
see, this is the thing. We've skipped installing so now this has a bunch of hard to debug assumptions in it. "passphrase for your SSH key"? what key?